mne/decoding/time_gen.py

```diff
@@ -702,7 +765,12 @@ def _predict(X, estimators, is_single_time_sample):
     # Compute prediction for each sub-estimator (i.e. per fold)
     # if independent, estimators = all folds
     for fold, clf in enumerate(estimators):
-        _y_pred = clf.predict(X)
+        if predict_function == 'predict':
+            _y_pred = clf.predict(X)
+        elif predict_function == 'predict_proba':
+            _y_pred = clf.predict_proba(X)
+        elif predict_function == 'decision_function':
+            _y_pred = clf.decision_function(X)
```

Comment:
Actually, couldn't the whole PR basically reduced to adding a setable default string attribute "predict_function" to the gat object that defaults to "predict", and then

```
_y_pred = getattr(clf, self.predict_function)(X)
```

So if you wanted probabilities, you'd do
`gat = GeneralizationAcrossTime(scorer=roc_auc, predict_method="predict_proba")`

Or
```
gat = GeneralizationAcrossTime(scorer=roc_auc)
gat.predict_method = "predict_proba"
gat.predict(epochs);
```

?